appveyor.yml

```diff
@@ -18,16 +18,33 @@ install:
 before_build:
   - cd ..
   - depot_tools\gclient config https://github.com/google/shaka-packager.git --name=src --unmanaged
+  - set GYP_DEFINES='target_arch="%PLATFORM%"'
   - depot_tools\gclient sync
 
 build_script:
   - cd src
-  - ..\depot_tools\ninja -C "out\%CONFIGURATION%" -k 100
-  - copy "out\%CONFIGURATION%\packager.exe" packager-win.exe
+  - if [%PLATFORM%] == [x64] (
+      ..\depot_tools\ninja -C "out\%CONFIGURATION%_x64" -k 100
```

Comment:
My first version is setting a new variable, but it doesn't work.
https://github.com/hchen1506/shaka-packager/blob/91b82d8fb3f741c70ef721d9abc69d84cf25920a/appveyor.yml

I guess there is scope problem for if-else...

This version works now. PTAL.